test/test.js

```diff
@@ -63,24 +64,27 @@ assert.equal(shell.error(), null);
 assert.equal(result, false);
 
 //link
-var result = shell.test('-d', 'resources/link');
-assert.equal(shell.error(), null);
-assert.equal(result, false);
-
-var result = shell.test('-f', 'resources/link');
-assert.equal(shell.error(), null);
-assert.equal(result, true);//true
-
-var result = shell.test('-L', 'resources/link');
-assert.equal(shell.error(), null);
-assert.equal(result, true);//true
-
-var result = shell.test('-L', 'resources/badlink');
-assert.equal(shell.error(), null);
-assert.equal(result, true);//true
-
-var result = shell.test('-L', 'resources/404');
-assert.equal(shell.error(), null);
-assert.equal(result, false);//false
+// Windows is weird with links so skip these tests
```

Comment:
I agree, if you could open a github issue for the bug, that would be nice to have documented. And if we can get that fixed, even better!